app/jobs/resolution_proofing_job.rb

```diff
@@ -11,7 +11,7 @@ class ResolutionProofingJob < ApplicationJob
   )
 
   def perform(result_id:, encrypted_arguments:, trace_id:, should_proof_state_id:,
-              dob_year_only:)
+              dob_year_only:, document_expired:)
```

Comment:
High level flow step 4: receiving `document_expired` here so we can skip AAMVA here (per ticket)